Remotion/Data/DomainObjects.ObjectBinding.UnitTests/BindableDomainObjectImplementationTest.cs

```diff
@@ -94,5 +96,13 @@ public void DisplayName_ViaImplementation_Overridden ()
       var implementation = BindableDomainObjectImplementation.Create (wrapper);
       Assert.That (implementation.DisplayName, Is.EqualTo ("TheDisplayName"));
     }
+
+    [Test]
+    public void CanAscribeTo_BindableDomainObjectImplementation_Mixin ()
+    {
+      Assert.IsTrue(
+          !typeof (BindableDomainObjectImplementation).CanAscribeTo (typeof (Mixin<,>)),
+          "we assume the mixin does not have a base object");
```

Comment:
Misunderstanding:
> Please is Is.True syntax in assertion.
> Please update to use extension method.

```suggestion
      Assert.IsTrue(
          !typeof (BindableDomainObjectImplementation).CanAscribeTo (typeof (Mixin<,>)),
          "we assume the mixin does not have a base object");
```
```suggestion
      Assert.That (typeof (BindableDomainObjectImplementation).CanAscribeTo (typeof (Mixin<,>)), Is.False);
```

I've also been able to update the descriptions for better context.